src/docs/no-more-than-100-fields.md

```diff
@@ -0,0 +1,353 @@
+# no-more-than-100-fields
+
+This rule flags entities with more than 100 fields. If a root entity query has more than 100 fields and requests over 200 records, the query is capped to 200 records.
+
+See [Feature Limitations of Offline GraphQL
```

Comment:
I'm not sure if we should include that reference specifically in this content, since I don't think the backing link actually describes any of these limitations.